### ISSUE TITLE
How do I start NoSleep when not invoked at Start-Up?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [irishuang1211/macosx-nosleep-extension#9]
```
I would like to be able to click on an Application or run a script to start 
NoSleep when it isn’t running.  The only way I know to to that is to set the 
NoSleep preference item to start up art start-up and than reboot the Mac.  
Seem’s like a lot of work to get this running again??

Thx for any help here!  This is a voice little program!  It allows me to walk 
around my office with my Mac talking on Skype with the lid closed!


```

Original issue reported on code.google.com by `corde...@gmail.com` on 8 Oct 2011 at 12:22

**Comment by GoogleCodeExporter** [comment]
```
There is a workaround for this - you can find NoSleepHelper.app in 
"/Library/Application Support/nosleep".
You also can create an alias in your /Application folder.

I'm bad in creating arts (including icons), so I have to hide it, cause I don't 
have an icon for it:(

```

Original comment by `integral...@gmail.com` on 8 Oct 2011 at 8:56
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3
```

Original comment by `integral...@gmail.com` on 23 Feb 2012 at 8:40
- Changed state: **Fixed**
